utils/common-messages.ts

```diff
@@ -15,10 +15,18 @@ export const commonMessages = defineMessages({
     id: 'tooltip.date-at-time',
     defaultMessage: '{date, date, long} at {time, time, short}',
   },
+  declineButton: {
+    id: 'button.decline',
+    defaultMessage: 'Decline',
+  },
```

Comment:
This probably shouldn't be commonised just yet and can stay as `auth.authorize.action.decline` in `authorize.vue`.